src/SFA.DAS.Authorization.EmployerFeatures/Context/AuthorizationContextExtensions.cs

```diff
@@ -4,10 +4,17 @@ namespace SFA.DAS.Authorization.EmployerFeatures.Context
 {
     public static class AuthorizationContextExtensions
     {
-        public static void AddEmployerFeatureValues(this IAuthorizationContext authorizationContext, long accountId, string userEmail)
+        public static void AddEmployerFeatureValues(this IAuthorizationContext authorizationContext, long? accountId, string userEmail)
         {
-            authorizationContext.Set(AuthorizationContextKey.AccountId, accountId);
-            authorizationContext.Set(AuthorizationContextKey.UserEmail, userEmail);
+            if (accountId.HasValue)
+            {
+                authorizationContext.Set(AuthorizationContextKey.AccountId, accountId);
+            }
```

Comment:
From memory, there is a slight difference between the AuthorizationContext returning a null value and it not finding a value. I'm not sure that will make a difference to the consuming code.